apps/SageAccountWeb/src/components/SageResourcesPage.tsx

```diff
@@ -5,15 +5,17 @@ import { useHistory } from 'react-router-dom'
 import { StyledOuterContainer } from './StyledComponents'
 import { BackButton } from './BackButton'
 import { useSourceAppConfigs } from './useSourceAppConfigs'
-import { useSourceApp } from './useSourceApp'
+import {
+  DEFAULT_SOURCE_APP_ID,
+  SourceAppProvider,
+  useSourceApp,
+} from './useSourceApp'
 
-export type SageResourcesPageProps = {}
-
-export const SageResourcesPage = (props: SageResourcesPageProps) => {
+export function SageResourcesPageInternal() {
   const history = useHistory()
   const theme = useTheme()
   const sourceAppConfigs = useSourceAppConfigs()
-  const sageSourceAppConfig = useSourceApp('SAGE')
```

Comment:
This is how we got a custom app config before, but it just updated branding components and did not update the theme, which is why I added the context object.